### PR TITLE
Updates postcss to the latest stable version

### PR DESCRIPTION
**Pull request by praveenmurugan101**

### CODE REVIEW

**Review by sainab-omar — APPROVED** [review]
Looks great 👍

**Review by televisedbrain — APPROVED** [review]

**Review by Hira-Naveed — APPROVED** [review]
Looking good 🥇

File: Dockerfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ RUN cd app
 RUN apt-get -y update
 
 # Install python3 
-RUN apt-get -y install python3.10
+RUN apt-get -y install python3 python3-pip
 
 ## Install pip
 RUN apt-get -y install curl
```
Comment (ellisrichardj) [inline Dockerfile:30]: Lines 30 and 31 shouldn't be required now as pip is installed with the apt command on line 27.

**Review by ellisrichardj — APPROVED** [review]
Marvellous - thanks for the additional changes 👍

**Pull request merged**